docs/DEPLOYMENT.md

```diff
@@ -116,22 +116,25 @@ installer image locally in a hardware lab. Note the changes made to both ```url`
 #!ipxe
 # set url https://github.com/lf-edge/eve/releases/download/snapshot/amd64.
 set url https://10.0.0.2/eve/releases/download/snapshot/amd64.
-# set eve_args eve_soft_serial=${ip} eve_reboot_after_install
-set eve_args eve_soft_serial=${ip} eve_install_server=zedcontrol.hummingbird.zededa.net eve_reboot_after_install
-
-# you are not expected to go below this line
 set console console=ttyS0 console=ttyS1 console=ttyS2 console=ttyAMA0 console=ttyAMA1 console=tty0
+set eve_args eve_soft_serial=${mac:hexhyp} eve_install_server=zedcontrol.hummingbird.zededa.net eve_reboot_after_install getty
```

Comment:
Why do we want to change the soft serial from using the IP address to using the MAC address? We support both but we used the IP in equinox metal

Reply:
We probably don't. I took a copy paste from the actual ipxe.cfg, although we have an IP based version as well. 

I'll fix that up.

Reply:
Updated